runtime/src/main/java/com/tietoevry/quarkus/resteasy/problem/ProblemRuntimeConfig.java

```diff
@@ -0,0 +1,46 @@
+package com.tietoevry.quarkus.resteasy.problem;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithName;
+
+@ConfigMapping(prefix = "quarkus.resteasy.problem")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface ProblemRuntimeConfig {
+
+    /**
+     * Config for ConstraintViolationException mapper
+     */
+    @WithName("constraint-violation")
+    ConstraintViolationMapperConfig constraintViolation();
+
+    interface ConstraintViolationMapperConfig {
+        static ConstraintViolationMapperConfig defaults() {
+            return new ConstraintViolationMapperConfig() {
+                @Override
+                public int status() {
+                    return 400;
+                }
+
+                @Override
+                public String title() {
+                    return "Bad Request";
```

Comment:
Maybe it's worth to define a constant to keep the default values aligned.

Reply:
Yeah, I didn't like this defaults() method that much from the very beginning, will try to simplify this.